sphinx_immaterial/apidoc/object_toc.py

```diff
@@ -143,6 +159,12 @@ def _collect(
                 if new_node is not None:
                     target += new_node
                     target = new_node
+            elif isinstance(source, docutils.nodes.rubric):
+                # Rubric.  Try to create synthetic section.
+                new_node = _make_section_from_rubric(source)
+                if new_node is not None:
+                    target += new_node
+                    target = new_node
```

Comment:
No need to assign to target, instead should return as in the `term` case before since rubrics cannot contain children (because they are just headings, not sections).  If you want them to be able to have chosen that would require more work.